js/ui/tree_list.d.ts

```diff
@@ -1184,6 +1192,36 @@ export default class dxTreeList extends Widget<dxTreeListOptions> implements Gri
     updateDimensions(): void;
 }
 
+type dxTreeListDefaultToolbarItemName = 'addRowButton' | 'applyFilterButton' | 'columnChooserButton' | 'revertButton' | 'saveButton' | 'searchPanel';
+
+/**
+ * @docid
+ * @type object
+ * @namespace DevExpress.ui
+ */
+export interface dxTreeListToolbarItem extends dxToolbarItem {
+  /**
+   * @docid
+   * @type Enums.TreeListToolbarItemName|string
+   * @public
+  */
+  name?: dxTreeListDefaultToolbarItemName | string
```

Comment:
Why is string type necessary here? DataGrid and TreeList have limited build-in items.

Reply:
It might be useful in case user wants to set custom name to identificate it later, for example, in `onToolbarPreparing` event. I wrote it following example of htmlEditor's toolbar: https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/html_editor.d.ts#L495